composer.json

```diff
@@ -48,34 +48,34 @@
         "php": "^7.4 || ~8.0.0 || ~8.1.0",
         "ext-mongodb": "*",
         "container-interop/container-interop": "^1.2.0",
-        "doctrine/doctrine-laminas-hydrator": "^3.0.1",
+        "doctrine/doctrine-laminas-hydrator": "^3.1.0",
         "doctrine/doctrine-module": "^5.2.1",
-        "doctrine/event-manager": "^1.1.1",
-        "doctrine/mongodb-odm": "^2.3.2",
-        "laminas/laminas-eventmanager": "^3.4.0",
-        "laminas/laminas-modulemanager": "^2.11.0",
-        "laminas/laminas-mvc": "^3.3.3",
+        "doctrine/event-manager": "^1.2.0",
+        "doctrine/mongodb-odm": "^2.4.2",
+        "laminas/laminas-eventmanager": "^3.5.0",
+        "laminas/laminas-modulemanager": "^2.12.0",
+        "laminas/laminas-mvc": "^3.3.5",
         "laminas/laminas-servicemanager": "^3.10.0",
-        "laminas/laminas-stdlib": "^3.7.1",
-        "symfony/console": "^5.4.5 || ^6.0.5",
-        "symfony/var-dumper": "^5.4.6 || ^6.0.6"
+        "laminas/laminas-stdlib": "^3.13.0",
+        "symfony/console": "^5.4.15 || ^6.0.15",
+        "symfony/var-dumper": "^5.4.14 || ^6.0.14"
     },
     "require-dev": {
-        "doctrine/coding-standard": "^9.0.0",
-        "doctrine/persistence": "^2.4.0 || ^3.0.0",
+        "doctrine/coding-standard": "^9.0.2",
+        "doctrine/persistence": "^2.5.5 || ^3.0.4",
         "jangregor/phpstan-prophecy": "^1.0.0",
-        "laminas/laminas-db": "^2.13.4",
-        "laminas/laminas-developer-tools": "^2.3.0",
-        "laminas/laminas-hydrator": "^4.3.1",
-        "laminas/laminas-i18n": "^2.14.0",
-        "laminas/laminas-log": "^2.15.0",
-        "laminas/laminas-serializer": "^2.12.0",
+        "laminas/laminas-db": "^2.15.0",
+        "laminas/laminas-developer-tools": "^2.5.0",
+        "laminas/laminas-hydrator": "^4.3.2",
+        "laminas/laminas-i18n": "^2.15.0",
+        "laminas/laminas-log": "^2.15.3",
+        "laminas/laminas-serializer": "^2.13.0",
         "laminas/laminas-session": "^2.12.1",
-        "laminas/laminas-view": "^2.19.1",
+        "laminas/laminas-view": "^2.20.0",
         "phpspec/prophecy-phpunit": "^2.0.1",
```

Comment:
Could this perhaps be dropped? :thinking:

Reply:
This can indeed be removed, as there is no direct dependency on laminas-view. However, it will be installed anyways:

```bash
$ composer why laminas/laminas-view
doctrine/doctrine-mongo-odm-module  4.2.x-dev  requires (for development)  laminas/laminas-view (^2.20.0)  
laminas/laminas-developer-tools     2.5.0      requires                    laminas/laminas-view (^2.13.1)
laminas/laminas-form                3.2.0      conflicts                   laminas/laminas-view (<2.14.0)  
laminas/laminas-i18n                2.15.0     conflicts                   laminas/laminas-view (<2.20.0)
laminas/laminas-mvc                 3.5.0      requires                    laminas/laminas-view (^2.14)
```

I a going to remove the dependency though, as we should only list direct dependencies. Code from the namespace `Laminas\View` is used nowhere in DoctrineMongoODMModule.

Reply:
It's mostly about reducing the dependency graph anyway :+1: